### ISSUE TITLE
Running the demo using v12.0.rc1 fails on Ubuntu

### ISSUE THREAD

**Issue by airnandez** [airnandez/lsst-cvmfs#7]
Running the stack demo on Ubuntu using `v12.0.rc1` does not work. This is the error message:

```
$ source /cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/loadLSST.bash
$ setup obs_sdss
$ ./bin/demo.sh 
Setting up: astrometry_net_data             Flavor: Linux64    Version: LOCAL:/home/ubuntu/lsst/lsst_dm_stack_demo-12.0.rc1/lsst_dm_stack_demo/astrometry_net_data
Traceback (most recent call last):
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/pipe_tasks/2016_01.0-48-g7bf2411/bin/processCcd.py", line 23, in <module>
    from lsst.pipe.tasks.processCcd import ProcessCcdTask
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/pipe_tasks/2016_01.0-48-g7bf2411/python/lsst/pipe/tasks/processCcd.py", line 22, in <module>
    from lsst.ip.isr import IsrTask
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/ip_isr/2016_01.0-16-g395f455/python/lsst/ip/isr/__init__.py", line 24, in <module>
    from .isrLib import *
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/ip_isr/2016_01.0-16-g395f455/python/lsst/ip/isr/isrLib.py", line 34, in <module>
    _isrLib = swig_import_helper()
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/ip_isr/2016_01.0-16-g395f455/python/lsst/ip/isr/isrLib.py", line 30, in swig_import_helper
    _mod = imp.load_module('_isrLib', fp, pathname, description)
  File "/cvmfs/lsst.in2p3.fr/software/linux-x86_64/lsst-v12.0.rc1/Linux64/base/2.2016.10-2-ge90c490+1/python/lsstimport.py", line 102, in imp_load_module
    module = orig_imp_load_module(name, *args)
ImportError: libssl.so.10: cannot open shared object file: No such file or directory
```

Note that this is a [known issue](https://jira.lsstcorp.org/browse/DM-5802) and the experts are investigating how to solve it.

I'll keep this issue open until a solution or turnaround is found.

**Comment by timj** [comment]
I don't think we are entirely agreed the best way to solve it. It's been debated for months now. Using the bundled version means you are only getting updates when MariaDB is updated. Using the system is better but the problem is that the conda SSL and the system SSL are not compatible. MariaDB locates the conda one but when linking happens later in the build none of the tools know where the conda libraries are and so attempt to link the system one which has a different version.

@jhoblitt, @mjuric and others need to agree on a solution that would work in conda and without conda.

**Comment by mjuric** [comment]
@timj, I'm happy to consult given I have some experience with conda, but I don't think I should be involved in _decisions_ at this level. This looks like a release management / dev support / software architecture issue.

**Comment by timj** [comment]
@ktlim decided on the approach to use for `mariadbclient`. I was asking you because you have clearly thought about the problem as I saw you had test branches on exactly this issue from your conda development. The release is a different issue.

**Comment by airnandez** [comment]
This issue is solved in `v12.0` which is online: this version, built on CentOS 7, works both on CentOS 7 and Ubuntu 14.

Thanks to the LSST software packagers for solving this issue upstream.
